src/components/PeopleFilters.tsx

```diff
@@ -1,21 +1,74 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+
+type Props = {
+  handleFilter: (value: string) => void,
+  handleCentury:(value: string) => void,
+  century: string[],
+  sex: string,
+  query: string
+  hendleQuery: (value: string) => void,
+};
+
+export const PeopleFilters:React.FC<Props> = ({
+  handleFilter,
+  handleCentury,
+  century,
+  sex,
+  query,
+  hendleQuery,
+}) => {
+  const { search } = useLocation();
+  const centuryArray = [16, 17, 18, 19, 20];
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <a
```

Comment:
why do you use a for links instead of ready-to-use component SearchLink ?

Consider refactor this logic. Also you don't need set click handler if you will use SearchLink component